SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/SpatialPackageMapClientMock.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "EngineClasses/AbstractPackageMapClient.h"
+#include "WorkerSDK/improbable/c_worker.h"
+
+#include "SpatialPackageMapClientMock.generated.h"
+
+UCLASS()
+class USpatialPackageMapClientMock : public UAbstractPackageMapClient
+{
+	GENERATED_BODY()
+public:
+
+	void Init(Worker_EntityId inReturnEntityId);
+
+	virtual Worker_EntityId GetEntityIdFromObject(const UObject* Object) override;
+
+private:
+	Worker_EntityId ReturnEntityId;
```

Comment:
What does return mean here? I'm not sure what this name is trying to tell me.

Reply:
Is it just that it returns it at some point? If so then just call it the EntityId.